[2024-08-26T04:48:23.422+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-26T04:48:23.463+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_and_transform_dag.run_spark_job manual__2024-08-26T04:48:18.534672+00:00 [queued]>
[2024-08-26T04:48:23.485+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_and_transform_dag.run_spark_job manual__2024-08-26T04:48:18.534672+00:00 [queued]>
[2024-08-26T04:48:23.486+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-26T04:48:23.521+0000] {taskinstance.py:2879} INFO - Executing <Task(SparkSubmitOperator): run_spark_job> on 2024-08-26 04:48:18.534672+00:00
[2024-08-26T04:48:23.530+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=673) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-26T04:48:23.531+0000] {standard_task_runner.py:72} INFO - Started process 674 to run task
[2024-08-26T04:48:23.532+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_and_transform_dag', 'run_spark_job', 'manual__2024-08-26T04:48:18.534672+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Dags.py', '--cfg-path', '/tmp/tmpsp38wbry']
[2024-08-26T04:48:23.536+0000] {standard_task_runner.py:105} INFO - Job 171: Subtask run_spark_job
[2024-08-26T04:48:23.638+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_and_transform_dag.run_spark_job manual__2024-08-26T04:48:18.534672+00:00 [running]> on host f00445136288
[2024-08-26T04:48:23.784+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_and_transform_dag' AIRFLOW_CTX_TASK_ID='run_spark_job' AIRFLOW_CTX_EXECUTION_DATE='2024-08-26T04:48:18.534672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-26T04:48:18.534672+00:00'
[2024-08-26T04:48:23.785+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-26T04:48:23.819+0000] {base.py:84} INFO - Retrieving connection 'SPARK-PROJECT'
[2024-08-26T04:48:23.820+0000] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark:7077 --conf spark.driver.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --conf spark.executor.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --name spark_transform_job --deploy-mode client /spark/app/transform.py
[2024-08-26T04:48:27.094+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SparkContext: Running Spark version 3.5.2
[2024-08-26T04:48:27.101+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SparkContext: OS info Linux, 6.6.26-linuxkit, amd64
[2024-08-26T04:48:27.102+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SparkContext: Java version 17.0.12
[2024-08-26T04:48:27.219+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-08-26T04:48:27.423+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceUtils: ==============================================================
[2024-08-26T04:48:27.424+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-08-26T04:48:27.425+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceUtils: ==============================================================
[2024-08-26T04:48:27.427+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SparkContext: Submitted application: DataCleaning
[2024-08-26T04:48:27.473+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-08-26T04:48:27.487+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceProfile: Limiting resource is cpu
[2024-08-26T04:48:27.488+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-08-26T04:48:27.603+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SecurityManager: Changing view acls to: ***
[2024-08-26T04:48:27.604+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SecurityManager: Changing modify acls to: ***
[2024-08-26T04:48:27.606+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SecurityManager: Changing view acls groups to:
[2024-08-26T04:48:27.608+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SecurityManager: Changing modify acls groups to:
[2024-08-26T04:48:27.609+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:27 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-08-26T04:48:28.008+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO Utils: Successfully started service 'sparkDriver' on port 37083.
[2024-08-26T04:48:28.048+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkEnv: Registering MapOutputTracker
[2024-08-26T04:48:28.094+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkEnv: Registering BlockManagerMaster
[2024-08-26T04:48:28.117+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-08-26T04:48:28.118+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-08-26T04:48:28.124+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-08-26T04:48:28.155+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-8812418b-67f3-4b82-9a0b-ad1e9d133fcd
[2024-08-26T04:48:28.179+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-08-26T04:48:28.206+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-08-26T04:48:28.428+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-08-26T04:48:28.525+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-08-26T04:48:28.614+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 ERROR SparkContext: Error initializing SparkContext.
[2024-08-26T04:48:28.615+0000] {spark_submit.py:579} INFO - org.apache.spark.SparkException: Could not parse Master URL: 'spark:7077'
[2024-08-26T04:48:28.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext$.org$apache$spark$SparkContext$$createTaskScheduler(SparkContext.scala:3194)
[2024-08-26T04:48:28.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:577)
[2024-08-26T04:48:28.617+0000] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-26T04:48:28.617+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-26T04:48:28.618+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-08-26T04:48:28.619+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-26T04:48:28.620+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-08-26T04:48:28.620+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-08-26T04:48:28.621+0000] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-26T04:48:28.622+0000] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-26T04:48:28.622+0000] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-26T04:48:28.623+0000] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-26T04:48:28.624+0000] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-26T04:48:28.624+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-26T04:48:28.625+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-26T04:48:28.626+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-26T04:48:28.626+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-08-26T04:48:28.635+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkUI: Stopped Spark web UI at http://f00445136288:4040
[2024-08-26T04:48:28.669+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-08-26T04:48:28.686+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO MemoryStore: MemoryStore cleared
[2024-08-26T04:48:28.687+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO BlockManager: BlockManager stopped
[2024-08-26T04:48:28.695+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-08-26T04:48:28.697+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 WARN MetricsSystem: Stopping a MetricsSystem that is not running
[2024-08-26T04:48:28.701+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-08-26T04:48:28.710+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO SparkContext: Successfully stopped SparkContext
[2024-08-26T04:48:28.711+0000] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-08-26T04:48:28.711+0000] {spark_submit.py:579} INFO - File "/spark/app/transform.py", line 135, in <module>
[2024-08-26T04:48:28.712+0000] {spark_submit.py:579} INFO - data_cleaner = DataCleaner("/opt/***/data/", snowflake_options)
[2024-08-26T04:48:28.713+0000] {spark_submit.py:579} INFO - ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-08-26T04:48:28.714+0000] {spark_submit.py:579} INFO - File "/spark/app/transform.py", line 10, in __init__
[2024-08-26T04:48:28.716+0000] {spark_submit.py:579} INFO - .getOrCreate()
[2024-08-26T04:48:28.717+0000] {spark_submit.py:579} INFO - ^^^^^^^^^^^^^
[2024-08-26T04:48:28.717+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 497, in getOrCreate
[2024-08-26T04:48:28.718+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 515, in getOrCreate
[2024-08-26T04:48:28.718+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 203, in __init__
[2024-08-26T04:48:28.719+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 296, in _do_init
[2024-08-26T04:48:28.719+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 421, in _initialize_context
[2024-08-26T04:48:28.719+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2024-08-26T04:48:28.720+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-08-26T04:48:28.720+0000] {spark_submit.py:579} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2024-08-26T04:48:28.721+0000] {spark_submit.py:579} INFO - : org.apache.spark.SparkException: Could not parse Master URL: 'spark:7077'
[2024-08-26T04:48:28.721+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext$.org$apache$spark$SparkContext$$createTaskScheduler(SparkContext.scala:3194)
[2024-08-26T04:48:28.721+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:577)
[2024-08-26T04:48:28.722+0000] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-26T04:48:28.722+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-26T04:48:28.722+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-08-26T04:48:28.723+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-26T04:48:28.723+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
[2024-08-26T04:48:28.723+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
[2024-08-26T04:48:28.724+0000] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-26T04:48:28.724+0000] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-26T04:48:28.724+0000] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-26T04:48:28.724+0000] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-26T04:48:28.724+0000] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-26T04:48:28.725+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-26T04:48:28.725+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-26T04:48:28.725+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-26T04:48:28.725+0000] {spark_submit.py:579} INFO - 
[2024-08-26T04:48:28.793+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO ShutdownHookManager: Shutdown hook called
[2024-08-26T04:48:28.794+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO ShutdownHookManager: Deleting directory /tmp/spark-a096f0d1-7b36-4247-8914-e79a5a1e37f8
[2024-08-26T04:48:28.805+0000] {spark_submit.py:579} INFO - 24/08/26 04:48:28 INFO ShutdownHookManager: Deleting directory /tmp/spark-2734a659-005d-46ef-9146-924180040a08
[2024-08-26T04:48:28.881+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark:7077 --conf spark.driver.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --conf spark.executor.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --name spark_transform_job --deploy-mode client /spark/app/transform.py. Error code is: 1.
[2024-08-26T04:48:28.902+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=extract_and_transform_dag, task_id=run_spark_job, run_id=manual__2024-08-26T04:48:18.534672+00:00, execution_date=20240826T044818, start_date=20240826T044823, end_date=20240826T044828
[2024-08-26T04:48:28.934+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-26T04:48:28.935+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 171 for task run_spark_job (Cannot execute: spark-submit --master spark:7077 --conf spark.driver.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --conf spark.executor.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --name spark_transform_job --deploy-mode client /spark/app/transform.py. Error code is: 1.; 674)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark:7077 --conf spark.driver.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --conf spark.executor.extraClassPath=/opt/bitnami/spark/jars/spark-snowflake_2.12-2.16.0-spark_3.2.jar --name spark_transform_job --deploy-mode client /spark/app/transform.py. Error code is: 1.
[2024-08-26T04:48:28.965+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-08-26T04:48:28.996+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-26T04:48:28.999+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
